public/app/features/alerting/unified/utils/access-control.ts

```diff
@@ -128,3 +129,12 @@ export function getRulesAccess() {
       contextSrv.hasPermission(provisioningPermissions.readSecrets),
   };
 }
+
+export const getCreateAlertInMenuAvailability = () => {
```

Comment:
I think this can just a Boolean, does not look like we need both booleans?

Oh, and a normal function is better than a const variable with lambda value (functions has a name and result in better call stacks)

Reply:
Again, you're right 👍 . That was needed in the previous code when we were adding the View alert list menu option. Thanks for catching it !

Reply:
updated!